### DISCUSSION TITLE
Omit camera switch for birdseye view

### DISCUSSION THREAD

**Discussion by mu71rd** [blakeblackshear/frigate#3021] [Ideas]
Hi,

I have several outdoor perimeter cameras, and a few indoor cameras that are only set to record when our alarm is set overnight, etc. The outdoor cameras record 24/7. Great so far!

I'd love to use birdseye view on a fully kiosk browser that's running on a tablet mounted in our kitchen, but I don't really need to take up screen space showing a view of myself looking at the tablet - ideally i would only see cameras that are triggered outdoors. 

Is there a way of omitting cameras from the birdseye view by default, or perhaps having a switch that would enable camera in/out of birdseye view. I can imagine wanting to see all triggered views at night if we're upstairs, or part-set on an alarm,  but I really don't want to see myself in the house during the day. 

even better would be to be able to put cameras into some kind of 'presence only' mode for indoor cameras - so that the average user cannot pull the feed up as a live view. This would take frigate close to the ultimate presence-detection option.

thanks for considering it 👍

**Comment by NickM-27** [comment]
There's a pinned issue for this https://github.com/blakeblackshear/frigate/issues/2582
